File: Src/WitsmlExplorer.Frontend/models/objectOnWellboreForSelection.ts
import { ContentTableRow } from "components/ContentViews/table";
import WellboreReference from "./jobs/wellboreReference";

export default interface ObjectOnWellboreForSelection extends ContentTableRow {
  objectType: string;
  logType: string;
  uid: string;
  name: string;
}

export interface ObjectsOnWellbore {
  wellboreReference: WellboreReference;
  selectedObjects: ObjectOnWellboreForSelection[];
}

File: Src/WitsmlExplorer.Api/Models/ObjectOnWellboreForSelection.cs
using Witsml.Data;

using WitsmlExplorer.Api.Jobs.Common.Interfaces;

namespace WitsmlExplorer.Api.Models
{
    public class ObjectOnWellboreForSelection
    {
        public string ObjectType { get; init; }
        public string LogType { get; init; }
        public string Uid { get; init; }
        public string Name { get; init; }
    }
}
